osmorg-editor.js

```diff
@@ -58,7 +58,7 @@ function queryForTags() {
                 tags.unshift(k + ' = ' + v);
             }
         }
-        textArea.value = tags.join('\n');;
+        textArea.value = tags.reverse().join('\n');;
```

Comment:
Wouldn't it be more efficient to replace `tags.unshift` above with a proper function?

Reply:
I though that `tags.unshift` did something else. We can just use `tags.push` instead and there won't be a need to use `.reverse()`.